source/manual/add-deployment-dashboard.html.md

```diff
@@ -66,6 +66,14 @@ Each dashboard can have parameters associated with it which effect how the dashb
 
 Params:
 
-- show_workers: This adds a row with workers failure and success panels, this is required for applications that have sidekiq workers. This is `false` by default.
+- has_workers: This adds a row with workers failure and success panels, this is required for applications that have sidekiq workers. This is `false` by default.
 
-- docs_name: This is the name of the application used in the developer documentation. Often the same as the repository name on Github. This deffaults to the `app_name`.
+- show_controller_errors: Adds a row which graphs 5XX responses broken down by Rails controller and action. For this graph to contain data, the application must report response statuses to Kibana as `@fields.status`, and also report the controller and action as `@fields.controller` and `@fields.action`. You may wish to set this to `false` for apps with very low request rates to avoid having an empty and potentially confusing graph. Defaults to `true`.
+
+- show_slow_requests: Adds a row which graphs response times broken down by Rails controller. For this graph to contain data, the application must report response times to Kibana as `@fields.duration` and the controller as `@fields.controller`. You may wish to set this to `false` for apps with very low request rates to avoid having an empty and potentially confusing graph. Defaults to `true`.
+
+- docs_name: This is the name of the application used in the developer documentation. Often the same as the repository name on Github. This defaults to the `app_name`.
+
+- error_threshold: The threshold at which application errors are highlighted in red. Defaults to 50.
+
+- warning_threshold: The threshold at which application errors are highlighted in orange. Defaults to 25.
```

Comment:
Would potentially be nice if these were alphabetically ordered.. but not a requirement. :)